### PR TITLE
Update framework assemblies list in assembly.c

### PR DESCRIPTION
**Pull request by akoeplinger**
Fixes https://github.com/mono/mono/issues/9973

### CODE REVIEW

**Comment by akoeplinger** [comment]
@monojenkins build pkg

**Comment by akoeplinger** [comment]
@monojenkins build pkg

File: mono/metadata/assembly.c [hunk 1]
```diff
@@ -254,8 +270,8 @@ static const AssemblyVersionMap framework_assemblies [] = {
 	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.Aes"),
 	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.ECDiffieHellman"),
 	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.ECDsa"),
-	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.Hashing"),
-	FACADE_ASSEMBLY ("System.Security.Cryptography.Encryption.Hashing.Algorithms"),
+	FACADE_ASSEMBLY ("System.Security.Cryptography.Hashing"),
+	FACADE_ASSEMBLY ("System.Security.Cryptography.Hashing.Algorithms"),
```
Comment (lambdageek) [inline mono/metadata/assembly.c]: what happened here?  Were the ones with `Encryption` in the name incorrect?

Comment (akoeplinger) [reply mono/metadata/assembly.c]: yup, it's the namespace, not the assembly name ;)

**Comment by akoeplinger** [comment]
@monojenkins build pkg

File: mono/metadata/assembly.c [hunk 2]
```diff
@@ -310,6 +339,7 @@ static const AssemblyVersionMap framework_assemblies [] = {
 	FACADE_ASSEMBLY ("System.Xml.XmlSerializer"),
 	FACADE_ASSEMBLY ("System.Xml.Xsl.Primitives"),
 	{"WindowsBase", 3},
+	{"cscompmgd", 0},
```
Comment (marek-safar) [inline mono/metadata/assembly.c]: I don't think we should add assemblies we know we'll never add/support

Comment (akoeplinger) [reply mono/metadata/assembly.c]: @marek-safar https://github.com/mono/mono/blob/master/mcs/class/Cscompmgd/Makefile 😄

Comment (marek-safar) [reply mono/metadata/assembly.c]: This was just one example, there are more like System.Web.RegularExpressions etc

Comment (akoeplinger) [reply mono/metadata/assembly.c]: I only added assemblies which we're building as part of net_4_x. I think we should add them all for consistency regardless of whether they are "useful".

**Pull request merged**